### ISSUE TITLE
Confirm OPTICAL_FLOW_RAD topic values

### ISSUE THREAD

**Issue by singhlok** [zehortigoza/aero-optical-flow#23]
Hi,

We have connected Lidar Lite v3 to Intel Aero RTF telemetry port. We can see the distance value under DISTANCE_SENSOR (current_distance) in MAVLINK Inspector. However, under the OPTICAL_FLOW_RAD section, the distance field always has a -1. Is this an expected behaviour?

I have attached the snapshot of MAVLINK Inspector
<img width="812" alt="screen shot 2018-04-15 at 09 46 16" src="https://user-images.githubusercontent.com/33088059/38776627-f422e006-4091-11e8-9cd6-c5d54c3570e8.png">

Also, can we use it in altitude mode to hover at a certain height?

**Comment by zehortigoza** [comment]
You can check in the code but yes in the OPTICAL_FLOW_RAD msg there is not valid distance, the I2C ports on top of Aero RTF is connected to AeroFC.

To use lidar readings in altitude mode you need to set EKF2_HGT_MODE = 2.

**Comment by rr2-ip** [comment]
Hi,

We have exactly the same issue and setup (except we are using the LeddarOne range finder).  
We had already set EKF2_HGT_MODE = 2, even after setting this parameter again (and rebooting the drone), the issue still remains.

Here is the output we got from aero-get-version.py:
BIOS_VERSION = Aero-01.00.13
OS_VERSION = Ubuntu 16.04.5 LTS"
AIRMAP_VERSION = unknown
PFGA_VERSION = 0xc2
AeroFC firmware version = 1.8.0

We have set SENS_EN_LEDDAR1 = 1, EKF2_AID_MASK = 2 and EKF2_HGT_MODE = 2.
The LeddarOne appears to be functioning correctly as we can see the distance under DISTANCE_SENSOR in MAVLink Inspector.

How can we get OPTICAL_FLOW_RAD to use the distance published by the LeddarOne?
